bundles/org.openhab.binding.senechome/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -48,15 +48,6 @@
 			<channel id="gridVoltagePhase3" typeId="gridVoltagePhase3"/>
 			<channel id="gridFrequency" typeId="gridFrequency"/>
 
-			<!-- SenecHomeStatistics -->
-			<channel id="liveBatCharge" typeId="liveBatCharge"/>
-			<channel id="liveBatDischarge" typeId="liveBatDischarge"/>
-			<channel id="liveGridImport" typeId="liveGridImport"/>
-			<channel id="liveGridExport" typeId="liveGridExport"/>
-			<channel id="liveHouseConsumption" typeId="liveHouseConsumption"/>
-			<channel id="livePowerGenerator" typeId="livePowerGenerator"/>
-			<channel id="liveEnergyWallbox1" typeId="liveEnergyWallbox1"/>
```

Comment:
Please provide upgrade instructions (XML file) to trigger an automatic removal of these channels (for existing things created with Main UI).

Reply:
@lolodomo what do you mean with this, do have an example or a link to the development docs?
Currently there is no replacement for this removed channels.

Reply:
You have an example here for example:
https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.danfossairunit/src/main/resources/OH-INF/update/instructions.xml
Note that you will have to add the property `thingTypeVersion` to your thing type.

Reply:
added instructions.xml and thingTypeVersion